File: AdvantageFramework/Database/Classes/AlertEmailRecipient.vb
Namespace Database.Classes

    <Serializable()>
    Public Class AlertEmailRecipient

#Region " Constants "



#End Region

#Region " Enum "

        Public Enum Properties

            RecipientAlertID
            RecipientID
            RecipientEmployeeCode
            RecipientEmailAddress
            RecipientEmployeeName
            EmployeeCode
            EmployeeEmail
            MailBeeTitle
            IsAssignee
            IsCC
            IsClientPortal
            IsCurrentRecipient
            SendEmail
            IsTaskEmployee

        End Enum

#End Region

#Region " Variables "



#End Region

#Region " Properties "

        Public Property RecipientAlertID As Integer = 0
        Public Property RecipientID As Integer = 0
        Public Property RecipientEmployeeCode As String = String.Empty
        Public Property RecipientEmailAddress As String = String.Empty
        Public Property RecipientEmployeeName As String = String.Empty
        Public Property EmployeeCode As String = String.Empty
        Public Property EmployeeEmail As String = String.Empty
        Public Property MailBeeTitle As String = String.Empty
        Public Property IsAssignee As Boolean = False
        Public Property IsCC As Boolean = False
        Public Property IsClientPortal As Boolean = False
        Public Property IsCurrentRecipient As Boolean = False
        Public Property SendEmail As Boolean = False
        Public Property IsTaskEmployee As Boolean = False

#End Region

#Region " Methods "

        Public Overrides Function ToString() As String

            ToString = Me.RecipientAlertID.ToString

        End Function

#End Region

    End Class

End Namespace

File: AdvantageFramework.Core/Database/Classes/AlertEmailRecipient.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AdvantageFramework.Core.Database.Classes
{
    public partial class AlertEmailRecipient
    {

        #region  Constants 



        #endregion

        #region  Enum 

        public enum Properties
        {
            RecipientAlertID,
            RecipientID,
            RecipientEmployeeCode,
            RecipientEmailAddress,
            RecipientEmployeeName,
            EmployeeCode,
            EmployeeEmail,
            MailBeeTitle,
            IsAssignee,
            IsCC,
            IsClientPortal,
            IsCurrentRecipient,
            SendEmail,
            IsTaskEmployee
        }

        #endregion

        #region  Variables 



        #endregion

        #region  Properties 

        public int RecipientAlertID { get; set; } = 0;
        public int RecipientID { get; set; } = 0;
        public string RecipientEmployeeCode { get; set; } = string.Empty;
        public string RecipientEmailAddress { get; set; } = string.Empty;
        public string RecipientEmployeeName { get; set; } = string.Empty;
        public string EmployeeCode { get; set; } = string.Empty;
        public string EmployeeEmail { get; set; } = string.Empty;
        public string MailBeeTitle { get; set; } = string.Empty;
        public bool IsAssignee { get; set; } = false;
        public bool IsCC { get; set; } = false;
        public bool IsClientPortal { get; set; } = false;
        public bool IsCurrentRecipient { get; set; } = false;
        public bool SendEmail { get; set; } = false;
        public bool IsTaskEmployee { get; set; } = false;

        #endregion

        #region  Methods 

        public override string ToString()
        {
            string ToStringRet = default;
            ToStringRet = RecipientAlertID.ToString();
            return ToStringRet;
        }

        #endregion

    }
}
